README.md

```diff
@@ -1,117 +1,69 @@
-  - **Core & difficult**: configurations that require a level of yak shaving to set up and maintain, such as very old versions of Linux (e.g. EL5) and very new tooling (e.g. XCode 6). This set of configurations may not be initially (and fully) available to the project as they may take time to set up in a reliable enough way.
+**[DigitalOcean](http://digitalocean.com/)**, a popular cloud hosting service, provide a significant amount of the resources required to run the Node.js project including key CI infrastructure and servers required to host [nodejs.org](https://nodejs.org/).
 
-* **Non-core platforms**: configurations that are secondary build targets of the io.js and libuv projects. These platforms are of interest to the community and core contributors will be given insight to the success of their commits on these platforms but failure will be reported as advisory-only. However, it is expected that project releases will pass on all of these platforms.
+**[Rackspace](https://www.rackspace.com/)**, a popular managed cloud company, provide significant resources used to power much of the Node.js project's CI system, including key Windows compilation servers, along with additional services such as object storage for backups via [Cloud Files](http://www.rackspace.com/en-au/cloud/files) and [Mailgun](http://www.mailgun.com/) for some [nodejs.org email](https://github.com/nodejs/email) services.
 
+### Tier-2 Providers
 
-### Configurations: All code
+The Node.js Foundation's tier-2 infrastructure providers fill essential gaps in architecture and operating system variations and shoulder some of the burden from the tier-1 providers, contributing to availability and speed in our CI system.
 
-All commits and pull requests will be submitted for compilation and execution of the **test-simple** test suite on the following platforms:
+![Tier 2 Infrastructure Providers](./provider-logos/tier-2-providers.png)
 
-  * Containerized 64bit image of the current oldest Ubuntu LTS Release (10.04, Lucid Lynx)
-  * Containerized 64bit image of the current latest Ubuntu LTS Release (14.04, Trusty Tahr)
+**[Microsoft Azure](https://azure.microsoft.com/en-us/)**, a cloud services platform, provide Windows test infrastructure for the Node.js CI system.
 
-### Configurations: Core & easy
+**[Joyent](https://www.joyent.com/)**, a public/private cloud infrastructure company, provide SmartOS test and build resources for the Node.js CI system.
 
-Additionally, all commits by core contributors will be submitted for compilation and execution of the **test-all** test suite on the following platforms:
+**[IBM](https://www.ibm.com/)**, via their cloud company, [SoftLayer](https://www.softlayer.com/) and the [Oregon State University Open Source Lab](https://osuosl.org/services/powerdev) provide PPC-based test and build infrastructure and other key hardware for testing and benchmarking for the Node.js project's CI system.
```

Comment:
I would split this up into softlayer and osuosl. Albeit both might be provided through IBM they have different agendas and both deserve their own spot.

softlayer provides us with general vm's for testing (and improved redundancy/latency) as well as baremetal for benchmarking. osuosl provides our ppc stack.